source/js/testdeleteUpdate.js

```diff
@@ -0,0 +1,5497 @@
+/* eslint-disable no-plusplus */
+// main.js
+
+// Import requried modules
+import * as storageFuncs from "./storage/storage.js";
+import * as fetcherFuncs from "./storage/fetcher.js";
+
+async function init() {
+  // TODO
+  // Load 10 recipes per category for explore page (offset=rand to get diff recipe everytime)
+  // Populate explore page cards
+  // add recipes to localstorage
+  // navigate to explore page
+
+  // fetch saved recipe cards from localstorage
+  // navigate to saved recipe page
+  // add save button action to every card and page to update the saved recipe page so we don't
+  // have to handle it when we open the page
+
+  // add event handler to search button to fetch recipes for query
+  // add recipes to localstorage
+  // populate search result cards
+  // navigate to search results page
+  console.log("in deleteUpdate.js");
+
+  const tempData = [
+    {
```

Comment:
this is a reallyyyy big object. maybe move to a file called "tempData.js" or something and just import it to keep things readable in this file?

Reply:
defintely